modules/subnets/main.tf

```diff
@@ -56,8 +56,9 @@ resource "google_compute_subnetwork" "subnetwork" {
     for_each = contains(keys(var.secondary_ranges), each.value.subnet_name) == true ? var.secondary_ranges[each.value.subnet_name] : []
 
     content {
-      range_name    = secondary_ip_range.value.range_name
-      ip_cidr_range = secondary_ip_range.value.ip_cidr_range
+      range_name              = secondary_ip_range.value.range_name
+      ip_cidr_range           = secondary_ip_range.value.ip_cidr_range
+      reserved_internal_range = secondary_ip_range.value.reserved_internal_range
```

Comment:
Lets bump the min provider version to 6.0 when support for this field was added to the provider (for this and beta module)

https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#600-august-26-2024